externs/shaka/player.js

```diff
@@ -1070,6 +1070,22 @@ shaka.extern.AdvancedAbrConfiguration;
  */
 shaka.extern.CmcdConfiguration;
 
+/**
+ * @typedef {{
+ *   enabled: boolean
+ * }}
+ *
+ * @description
+ *   Decoding for MPEG-5 Part2 LCEVC.
+ *
+ * @property {boolean} enabled
+ *   If <code>true</code>, enable LCEVC data to be passed to LCEVC Dil and
+ *   decode frames on a canvas element.
+ *   Defaults to <code>false</code>.
```

Comment:
Wouldn't you prefer this default to `true`?  It's gated by the inclusion of your node module, so enabling it by default should still be equivalent to disabling it by default for those who don't include the module.

For that matter, why have this config at all?  An app could include or exclude the module instead.  Is there any case where someone would want to turn it on and off per-playback?

Reply:
As per the suggestion we have removed the config option for LCEVC completely and now it will be activated based on the availability of the library on the page.